ajax/libs/tiny-date-picker/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "tiny-date-picker",
+  "description": "A small, dependency-free date picker",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/chrisdavies/tiny-date-picker.git"
+  },
+  "keywords": [
+    "date",
+    "picker",
+    "date",
+    "chooser",
+    "calendar"
+  ],
+  "author": "Chris Davies <ferociousturtle@gmail.com>",
+  "license": "MIT",
+  "homepage": "https://github.com/chrisdavies/tiny-date-picker#readme",
+  "autoupdate": {

```

Comment:
There are 16 releases on NPM but 14 on GitHub. Let's use NPM auto-update config instead of git auto-update. You can take a look at https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md